src/editorial/web/components/button/index.tsx

```diff
@@ -0,0 +1,266 @@
+import { css } from '@emotion/react';
+
+import { Design, Display, Format, Pillar, Special } from '@guardian/types';
+import {
+	Button as CoreButton,
+	ButtonProps as CoreButtonProps,
+	LinkButton as CoreLinkButton,
+	LinkButtonProps as CoreLinkButtonProps,
+	ButtonPriority,
+} from '@guardian/src-button';
+import {
+	culture,
+	labs,
+	lifestyle,
+	news,
+	opinion,
+	specialReport,
+	sport,
+	neutral,
+} from '@guardian/src-foundations';
+
+export interface ButtonProps extends CoreButtonProps {
+	format?: Format;
+	priority?: ButtonPriority;
+	children: React.ReactNode;
```

Comment:
Do we need to specify `priority` and `children` if they're already in `CoreButtonProps`?

Reply:
Good spot, shalll remove